src/software/ai/hl/stp/tactic/crease_defender/crease_defender_tactic_test.cpp

```diff
@@ -26,15 +26,15 @@ class CreaseDefenderTacticTest
     TbotsProto::AiConfig ai_config;
 };
 
-// TODO (#2512): Due to friction in ErForceSim, the ball does not reach defender
-TEST_F(CreaseDefenderTacticTest, DISABLED_test_chip_ball)
+TEST_F(CreaseDefenderTacticTest, test_chip_ball)
 {
-    Point enemy_threat_point = Point(-1.5, 0.5);
+    Point enemy_threat_point = Point(-0.5, 0);
     TbotsProto::CreaseDefenderAlignment alignment =
         TbotsProto::CreaseDefenderAlignment::CENTRE;
 
-    Point initial_position = Point(-3, 1.5);
-    BallState ball_state(enemy_threat_point, Vector(-2, 0));
+    Point initial_position = Point(-3, 1.0);
+
+    BallState ball_state(enemy_threat_point, Vector(-2.5, 0));
```

Comment:
i suggest changing this test scenario because in the current implementation, the `enemy_threat_point`, which normally tracks the ball and is updated every AI tick, is constant throughout the test, leading to the robot to not block/chip the ball. I have changed it so that the robot is already at the right block point in order to test that the chipping works. 

If we want to test the other part of this scenario - that the robot moves to the right position to block a moving ball, there are a couple of things we can do: 
- make the `crease_defender_tactic` take into account the trajectory of the enemy threat (which in this case is the ball), so that it can position itself to block in the absence of updates
- find a way to call `tactic->updateControlParams` to pass in new ball positions. I tested this in a hacky way by wrapping it in a validation function and the original test passes, but it could be easily added to the test fixture.

Reply:
Could you make an issue for this? I'll approve this PR but just make an issue for it